CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.0.1] - 2024-05-14
+
+Props: @GaryJones, @jrnfl, @terriann, @rebeccahum
+
+Increases requirements for PHPCS from 3.7.2 to 3.9.2 for improved PHP 8.2 support. Please ensure you run `composer update automattic/vipwpcs --with-dependencies` to benefit from this.
+
+### Added
+- [#805](https://github.com/Automattic/VIP-Coding-Standards/pull/805): Use PHPCSStandards fork due to squizlabs repo being abandoned.
```

Comment:
I'd move this to "Maintenance" and make the description more about documentation. Originally, it was expected that we'd need to switch to a different Composer package (hence the PR title), but somewhere down the line and before the PR was merged, it became clear that would not be needed, so it became a documentation/link change only.